cmd/update-rules/main.go

```diff
@@ -141,11 +162,11 @@ func UpdateRules(rules *config.RepositoryRules, branch, goVer string) {
 			continue
 		}
 
+		var branchRuleExists bool
+		// if the target branch rules already exists, update it
 		// update the rules for branch and its dependencies
 		updateBranchRules(&newBranchRule, branch, goVer)
 
-		var branchRuleExists bool
-		// if the target branch rules already exists, update it
```

Comment:
This diff is not needed.

Reply:
This is not resolved.

Reply:
based on the variable `branchRuleExists` the code runs 
```		
if !branchRuleExists {
			r.Branches = append(r.Branches, newBranchRule)
		}```

Reply:
What I meant was this diff is not needed. Earlier `branchRuleExists` was defined  after the `updateBranchRules()` func. We can still have it  there.